.github/workflows/pester.yml

```diff
@@ -1,15 +1,14 @@
-name: MacOS_Pester
+name: Pester Tests
 
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+on: [push]
```

Comment:
@MKletz I recommend that you do this on a `pull request`, and not on `push`. That is going to work out better for open source development where most (all?) of the work will be done via a pull request. Do you plan to do work outside of a pull request?

Reply:
nope, good catch. Fixed